doc/nrf/glossary.rst

```diff
@@ -0,0 +1,413 @@
+   For example, it has been used in sensors communicating to a broker via satellite link, over occasional dial-up connections with healthcare providers, and in a range of home automation and small device scenarios.
+
+Microcontroller Unit (MCU)
+   A small computer on an integrated circuit.
+
+Menuconfig
+   A tool to configure kernel source.
+   It was initially created for the Linux kernel.
+   It uses the Kconfig configuration language.
+
+Multicast addressing
+   An addressing type that uses a one-to-unique many association, where datagrams are routed from a single sender to multiple selected endpoints simultaneously in a single transmission.
+
+Narrowband Internet of Things (NB-IoT)
+   A narrowband technology standard with longer range, lower throughput, and better penetration in, for example, cellars and parking garages compared to LTE-M.
+   NB-IoT is most suitable for static, low throughput applications like smart metering, smart agriculture, and smart city applications.
+   Also known as Cat-NB1.
+
+Near Field Communication (NFC)
+   A standards-based short-range wireless connectivity technology that enables two electronic devices to establish communication by bringing them close to each other.
+
+Network Co-Processor (NCP)
+   A co-processor offloading network functions from the host processor.
+   In the |NCS| context, it is typically used in OpenThread and Zigbee platform designs.
+
+Network Time Protocol (NTP)
+   A networking protocol for clock synchronization between computer systems over packet-switched, variable-latency data networks.
+
+NFC-A Listen Mode
+   The initial mode of an NFC Forum Device when it does not generate a carrier.
+   The device listens for the remote field of another device.
+   See Near Field Communication (NFC).
+
+Noise Factor (NF)
+   The relation of the Signal-to-Noise Ratio (SNR) in the device input to the SNR in the device output.
+
+Non-volatile Memory Controller (NVMC)
+   A controller used for writing and erasing the internal flash memory and the User Information Configuration Registers (UICR).
+
+nRF Connect for Cloud
+   Nordic Semiconductor's platform for connecting IoT devices to the cloud, viewing and analyzing device message data, prototyping ideas that use Nordic Semiconductor chips, and more.
+   It includes a public REST API that can be used for building IoT solutions.
+   See `nRF Connect for Cloud`_.
+
+nRF repository
+   An |NCS| repository that does not have an externally maintained, open-source upstream.
+   It is exclusive to Nordic development.
+
+nRF Secure Immutable Bootloader (NSIB)
+   A first-stage secure bootloader built and maintained by Nordic Semiconductor.
+
+OpenAMP
+   A framework that provides software components that enable the development of software applications for Asymmetric Multiprocessing (AMP) systems.
+   See `OpenAMP`_.
+
+OpenThread
+   A portable and flexible open-source implementation of the Thread networking protocol.
```

Comment:
Add a link to openthread.io? (Already in links.txt)

Reply:
fixed